browser/download/brave_download_item_model.h

```diff
@@ -6,7 +6,7 @@
 #ifndef BRAVE_BROWSER_DOWNLOAD_BRAVE_DOWNLOAD_ITEM_MODEL_H_
 #define BRAVE_BROWSER_DOWNLOAD_BRAVE_DOWNLOAD_ITEM_MODEL_H_
 
-#include "chrome/browser/download/download_item_model.h"
+#include "chrome/browser/download/download_ui_model.h"
```

Comment:
we could replace it with a forward declaration

Reply:
Oops! Fixed now, thanks!

Additionally, making the most that I'm adding two items to the two lists at the bottom of //brave/browser/sources.gni, I've added one minor commit on top of this PR that normalizes & sorts those lists, which will make things easier for future additions to the file.

Will merge once CI confirms it's happy.